easybuild/easyconfigs/s/Score-P/Score-P-3.0-CrayGNU-2016.11-cuda-8.0.54.eb

```diff
@@ -0,0 +1,68 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013-2016 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'Score-P'
+version = '3.0'
+versionsuffix = '-cuda-8.0.54'
+
+homepage = 'http://www.score-p.org'
+description = """The Score-P measurement infrastructure is a highly scalable and
+ easy-to-use tool suite for profiling, event tracing, and online analysis of HPC
+ applications."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+
+source_urls = ['http://www.vi-hps.org/upload/packages/scorep/']
+sources = ['scorep-%(version)s.tar.gz']
+checksums = [ '44da8beaa3f71436a5f6fe51938aab2f' ]
+
+dependencies = [
+    ('libiberty', '5.3.0-scorep', '', True),
+    ('libunwind', '1.1-scorep', '', True),
+    ('Cube', '4.3.4', '', True),
+    ('OPARI2', '2.0.1', '', True),
+#PR3501    ('OTF2', '2.0', '', True),
+    ('SIONlib', '1.7.1', '', True),
+    ('PDT', '3.23', '', True),
+    ('papi/5.5.0.2', EXTERNAL_MODULE),
+    ('vampir/9.2.0', EXTERNAL_MODULE),
+    ('craype-accel-nvidia60', EXTERNAL_MODULE),
+    #   cudatoolkit/8.0.54_2.2.8_ga620558-2.1
+    #   cray-libsci_acc/16.11.1
+]
+
+configopts  = ' --with-machine-name=daint'
+configopts  = ' --enable-shared'
+configopts += ' --with-libpmi-include=/opt/cray/pe/pmi/default/include'
+configopts += ' --with-libpmi-lib=/opt/cray/pe/pmi/default/lib64'
+configopts += ' --with-librca-include=/opt/cray/rca/default/include'
+configopts += ' --with-librca-lib=/opt/cray/rca/default/lib64'
+# --- cuda
+configopts += ' --enable-cuda'
+configopts += ' --with-libcuda-lib=$CRAY_CUDATOOLKIT_DIR/lib64/stubs'
+configopts += ' --with-libcuda-include=$CRAY_CUDATOOLKIT_DIR/include'
+configopts += ' --with-libcudart=$CRAY_CUDATOOLKIT_DIR'
+configopts += ' --with-cupti=$CRAY_CUDATOOLKIT_DIR/extras/CUPTI'
+
+sanity_check_paths = {
+    'files': ["bin/scorep", "include/scorep/SCOREP_User.h",
+             ("lib/backend/libscorep_adapter_mpi_event.a",
+              "lib/libscorep_adapter_mpi_event.%s" % SHLIB_EXT) ],
+    'dirs': [],
+}
+
+# Ensure that local metric documentation is found by Cube GUI
+modextrapaths = { 'CUBE_DOCPATH': 'share/doc/scorep/profile' }
+modextravars = { 'CRAYPE_LINK_TYPE' : 'dynamic' }
```

Comment:
Dynamic builds are the default with EasyBuild, you could remove this. This variable also set if cudatoolkit is loaded.